lib/bank_scrap.rb

```diff
@@ -14,4 +14,6 @@ module BankScrap
   autoload :Bankinter,  'bank_scrap/banks/bankinter'
   autoload :Bbva,       'bank_scrap/banks/bbva'
   autoload :Ing,        'bank_scrap/banks/ing'
+  autoload :Openbank,   'bank_scrap/banks/openbank'
+  autoload :OpenbankAccount,   'bank_scrap/banks/openbank/account'
```

Comment:
Seeing this I realize that maybe we should define a new module for each bank, (instead of a class inheriting from `Bank`), that way we can use autoload to require the whole module. But I am not sure yet.